client/app/controllers/collection/index.js

```diff
@@ -93,6 +94,9 @@ export default Ember.Controller.extend({
         },
         changeRoute(path){
             this.transitionToRoute(path);
+        },
+        changeView(viewName) {
+            this.set('view', viewName);
```

Comment:
I'm not sure if we will add more types of views, so I can change it to toggle a `cardView` property